extraction_tools_wiki/wiki_page_titles.py

```diff
@@ -43,26 +42,24 @@ def __init__(self, base_url: str, categories: list, user_agent: str, user_email:
             'User-Agent': user_agent,
             'From': user_email
         }
-        self.page_titles = dict()  # type: dict
+        self.page_titles: Dict[str, str] = dict()
 
-    def __iter__(self):
+    def __iter__(self) -> str:

```

Comment:
So the `__iter__` actually is returning a generator of strings.

```python
from typing import Generator
def __iter__(self) -> Generator[str, None, None]:
    ...
```

Reply:
Awesome, thanks. I was thinking what I did was not correct.